test/src/edu/stanford/nlp/process/WhitespaceTokenizerTest.java

```diff
@@ -98,17 +98,6 @@ public void testPureWhiteSpaceEOLTokenizer() {
     }
   }
 
-  /* Tests the "no whitespace at all" bound */
```

Comment:
I see why the WhitespaceTokenizerTest showed up earlier.  Definitely could be squashed into the previous changes